cmds/ls/fileinfo_linux.go

```diff
@@ -125,4 +137,9 @@ func (fi longStringer) String() string {
 		fi.size,
 		fi.modTime.Format("Jan _2 15:04"),
 		fi.comp.String())
+
+	if fi.mode&os.ModeType == os.ModeSymlink {
+		s += fmt.Sprintf(" -> %v", fi.symlink)
```

Comment:
yeah, but ... it's a utf8 world. We don't escape the filenames either. For another day.